<p align="center">
  <h3 align="center">ðŸ”¬ MetaG Analysis - 16S Sequencing Pipeline </h3>
A repository for data of a bioinformatics project on the 16S sequencing. It includes a pipeline using QIIME2 to process data and do statistical analysis. 
</p>

---
## Authors 
A group of 4 students from ESTBarreiro with their school id number assigned.
- Aleff Cavalcante, 202300054
- Bianca Silva, 202300273
- Filipa Fernandes, 202300218
- Ravi Silva, 202100191

---
## Index


---
## Introduction
In here you can find our results on the study of 16S sequencing, based on this original paper: https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0298002.
Besides it, you can find a pipelinemade to recreate some of the steps of this paper, using QIIME2.

---
## Features


---
## Installation

### Pre Requisites
- Docker and QIIME2 image (there's more alternatives to it, but we advise this one)
- R with the packages phyloseq and decontam

### Installation Steps

1. Clone the repository:
   ```sh
   git clone https://github.com/AceSCav/MetaG_Analysis.git
   cd MetaG_Analysis/Pipeline
   Â´Â´Â´

---
## Usage
### Run the pipeline:
```sh
./Main.sh
```

After running you'll be prompt with the informations of this pipeline along side of how to use it. During the process you'll always be told what to do and explained what is happenning. You'll have the choice to skip some steps too.

### File Organization
